go.mod

```diff
@@ -1,20 +1,22 @@
 module github.com/databricks/cli
 
-go 1.23
+go 1.23.0
 
-toolchain go1.23.2
+toolchain go1.23.3
```

Comment:
Can you break this out into a separate PR?

Version bump also needs to update the GH actions.

Reply:
https://github.com/databricks/cli/pull/2038